### ISSUE TITLE
in thunderbird nvda does not announce pasted link in new message

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#15174]
### Steps to reproduce:
create a new email message in tunder bird.
walk with tabe to the field
message body.
now copy any link, in my tests i copied a file share link on one drive.
paste this link in this message.
now navigate with the up and down arrows through the text field.
### Actual behavior:
nvda announces only blank.

### Expected behavior:
nvda should announce the pasted link.
if you walk with ctrl + arrows you will eventually find the link and the link preview text, after that you can find the link by walking with the arrows like this and below as expected before.

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-28712,9157916f

#### Windows version:
11 22621.2066
#### Name and version of other software in use when reproducing the issue:
thunderbird.exe, Thunderbird 116.0
maybe playable since version 115

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[tunderbird.txt](https://github.com/nvaccess/nvda/files/12134965/tunderbird.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this issue with NvDA 2024.4 Beta and Thunderbird 128.2.0?

**Comment by fernando-jose-silva** [comment]
no, I have this problem again, with everything updated, thanks.

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this issue with NVDA 2025.3 and Thunderbird 144 or 140 ESR?

**Comment by fernando-jose-silva** [comment]
I can't play it anymore. Thanks.

**Comment by seanbudd** [comment]
Closing as fixed
